app/components/autocomplete/at_mention_group/at_mention_group.tsx

```diff
@@ -81,8 +88,12 @@ const GroupMentionItem = ({
                     style={style.rowIcon}
                 />
             </View>
-            <Text style={style.rowUsername}>{`@${completeHandle} - `}</Text>
-            <Text style={style.rowFullname}>{`${completeHandle}`}</Text>
+            <View
+                style={style.rowInfo}
+            >
+                <Text style={style.rowUsername}>{`@${completeHandle} - `}</Text>
+                <Text style={style.rowFullname}>{`${completeHandle}`}</Text>
```

Comment:
Why show it duplicate with and without the `@` sign?

Reply:
Whoops - fixed to use name + display name.